packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -17,32 +24,30 @@ describe('WorkspaceContext', () => {
   let getUsernameStub: jest.SpyInstance;
   let getUsernameOrAliasStub: jest.SpyInstance;
   let workspaceContextUtil: any; // TODO find a better way
-  let authUtil: any;
 
-  const mockWatcher = {
-    onDidChange: jest.fn(),
-    onDidCreate: jest.fn(),
-    onDidDelete: jest.fn()
-  };
+  let mockWatcher: { onDidChange: any; onDidCreate: any; onDidDelete: any; };
   let mockFileSystemWatcher: jest.SpyInstance;
+  let reloadConfigAggregatorsMock: jest.SpyInstance;
 
   beforeEach(async () => {
+    mockWatcher = {
+      onDidChange: jest.fn(),
+      onDidCreate: jest.fn(),
+      onDidDelete: jest.fn()
+    };
+    reloadConfigAggregatorsMock = jest.spyOn(ConfigAggregatorProvider.prototype, 'reloadConfigAggregators');
+
     mockFileSystemWatcher = (vscode.workspace
       .createFileSystemWatcher as any).mockReturnValue(mockWatcher);
 
     const context = {
       subscriptions: []
     };
+    getUsernameOrAliasStub = (authUtilMock.prototype.getDefaultUsernameOrAlias as any).mockReturnValue(testAlias);
+    getUsernameStub = (authUtilMock.prototype.getUsername as any).mockReturnValue(testUser);
+    authUtilMock.getInstance.mockReturnValue(new AuthUtil());
```

Comment:
So for the modules where `mock` is used we can just referenced the typed mock value directly instead of spying on individual methods.   Note for the last line I have to set a return value for getInstance  and it returns a mocked authUtil instance. Could have done the same for the config agg but since it was a single method I went the simple route of just spying on the single method.